### PR TITLE
Cloud Tutorial for changing Domain to Discovery Server

### PR DESCRIPTION
**Pull request by jparisu**
Signed-off-by: jparisu <javierparis@eprosima.com>

### CODE REVIEW

File: docs/resources/tutorials/cloud/change_domain_to_ds/change_domain_client.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - name: ROS_2_Domain_0
+    kind: local
+    domain: 0
+
+  - name: ROS_2_ds_client
+    kind: local-discovery-server
+
+    discovery-server-guid:
+      ros-discovery-server: true
+      id: 2
```
Comment (rsanchez15) [inline docs/resources/tutorials/cloud/change_domain_to_ds/change_domain_client.yaml:14]: Then, we can use `id: 1` here

```suggestion
      id: 1
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]
I've already update the images to match this suggestions. Please update these images.

File: docs/rst/tutorials/cloud/change_domain_discovery_server/change_domain_discovery_server.rst [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+This indicates that the |rosrouter| has started correctly and it is currently running.
+
+.. code-block:: bash
+
+    Starting DDS Router Tool execution.
+    DDS Router running.
+
+In order to close the execution, just press ^C or send a signal (:code:`SIGINT 2` or :code:`SIGTERM 15`) to close it.
+
+Deploy ROS 2 Router as Discovery Server
```
[the remark this answers is not in the document]
Comment (jparisu) [reply docs/rst/tutorials/cloud/change_domain_discovery_server/change_domain_discovery_server.rst:189]: @juanlofer-eprosima ?

[the remark this answers is not in the document]
Comment (juanlofer-eprosima) [reply docs/rst/tutorials/cloud/change_domain_discovery_server/change_domain_discovery_server.rst:189]: I like @rsanchez15's suggestion, maybe add clarification: "Alternative deployment: configuring ROS 2 Router as Discovery Server's server"

File: docs/rst/tutorials/cloud/change_domain_discovery_server/change_domain_discovery_server.rst [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+.. _tutorials_router_change_domain_to_discovery_server_discovery_server:
+
+Discovery Server
+""""""""""""""""
+
+Some networks do not support multicast, or are so big that discovering the whole network implies high amount of time.
+In this cases, a new *Discovery Mechanism* specific of |fastdds| can be used, called **Discovery Server Discovery Mechanism**.
+This mechanisms uses a *Server* or network of *Servers* that centralize the discovery traffic and distributes it to only the required entities, while the communication still occurs *Peer to Peer*.
+In order to know more about *Discovery Server*, please refer to `Fast DDS Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery_server.html>`__ or check :ref:`this tutorial <internal ref>` about how to configure and use *Discovery Server*.
```
Comment (rsanchez15) [inline docs/rst/tutorials/cloud/change_domain_discovery_server/change_domain_discovery_server.rst:59]:
```suggestion
In order to know more about *Discovery Server*, please refer to `Fast DDS Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery_server.html>`__ or check :doc:`this tutorial <../../../../ros2_documentation/source/Tutorials/Advanced/Discovery-Server/Discovery-Server>` about how to configure and use *Discovery Server*.
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

**Review by rsanchez15 — APPROVED** [review]
LGTM

**Pull request merged**